tests/cypress/integration/reportRepair/communalMvp.spec.js

```diff
@@ -10,6 +11,11 @@ describe('communal (MVP)', () => {
     context('Behaviour', () => {
       context('When a user selects an option', () => {
         beforeEach(navigateToCommunalPage);
+        context('When a user selects: Yes', ()=>{
+          it('should redirect them to postcode page',  () => {
```

Comment:
Should this be `it should redirect them to the not eligible page` instead?

Reply:
Absolutely! Copy-paste gone wrong! 😳

Corrected now.